src/main.js

```diff
@@ -76,7 +76,7 @@ require.config({
 
 var templateEngine;
 require([
-  'jquery', 'TemplateEngine'
+  'jquery', 'lib/TemplateEngine'
```

Comment:
Yep should work as configured, but it doesn't. Without this change the "compiled" release cannot include the TemplateEngine.